src/app.jsx

```diff
@@ -17,9 +11,9 @@ import PlaylistList from "./components/playlist/playlist-list";
 import PlaylistEdit from "./components/playlist/playlist-edit";
 import PlaylistCreate from "./components/playlist/playlist-create";
 import MediaList from "./components/media/media-list";
+import SlidePreview from "./components/slide/preview/slide-preview";
+// import MediaEdit from "./components/media/media-edit";
```

Comment:
Remove

Reply:
We agreed to keep this until the api is updated, I can remove it, but should i remove the files as well? Will the api not have the possibility to edit media added?

Reply:
Commented out code should just be removed. You can leave the component alone if you like :)